src/dotnet/CarbonAware.Aggregator/src/IAggregator.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CarbonAware.Aggregator
```

Comment:
Everything in this namespace is just scaffolding for the aggregator, no specific implementation